src/ui/Analytics/Analytics.ts

```diff
@@ -158,7 +161,9 @@ export class Analytics extends Component {
      * Default value is `undefined`, and the value of this parameter will fallback to the organization used for the
      * search endpoint.
      */
-    organization: ComponentOptions.buildStringOption()
+    organization: ComponentOptions.buildStringOption(),
+    populateDataLayer: ComponentOptions.buildBooleanOption({ defaultValue: true }),
```

Comment:
I'm not certain the name "populateDataLayout" is clear enough ?

How about having some kind of naming that explicitly mention google tag manager ?

Would also be interesting to see the accompanying documentation for that option

Reply:
I'll go with:

`pushToGtmDataLayer` and `gtmDataLayerName`

Documentation incoming :)